test/dotnet/ExpectedOutput/TestIsolationLevel.out

```diff
@@ -0,0 +1,48 @@
+#Q#SELECT set_config('babelfishpg_tsql.enable_repeatable_read_isolation_level','on',false)
+#D#text
+on
+#Q#SELECT set_config('babelfishpg_tsql.enable_serializable_isolation_level','off',false)
+#D#text
+off
+#Q#select @@trancount;
+#D#int
+1
+#Q#select current_setting('transaction_isolation')
+#D#text
+read committed
```

Comment:
why is this read committed, ideally both transaction and default isolation levels should be same. Could you please take a look?

Reply:
txn#!#begin#!#isolation#!#rr
This is equivalent to first begin transaction then set isolation level
hence we only have default_transaction_isolation set to rr

Reply:
If i run set transaction isolation level reapeatable read
and then begin transaction 
both transaction_isolation & default are set to repeatable read